Symfony/CS/AbstractFixer.php

```diff
@@ -21,22 +21,9 @@
     /**
      * {@inheritdoc}
      */
-    public function getLevel()
+    public function configure(array $configuration = null)
     {
-        static $map = array(
-            'PSR1' => FixerInterface::PSR1_LEVEL,
-            'PSR2' => FixerInterface::PSR2_LEVEL,
-            'Symfony' => FixerInterface::SYMFONY_LEVEL,
-            'Contrib' => FixerInterface::CONTRIB_LEVEL,
-        );
-
-        $level = current(explode('\\', substr(get_called_class(), strlen(__NAMESPACE__.'\\Fixer\\'))));
-
-        if (!isset($map[$level])) {
-            throw new \LogicException(sprintf('Can not determine Fixer level: "%s".', $level));
-        }
-
-        return $map[$level];
+        return;
```

Comment:
Most of our fixers don't need configuration - thus, in our abstract fixer, this method is empty to not force most of fixers implement it empty themselves